.conda/meta.yaml

```
@@ -1,43 +1,35 @@
 package:
   name: rbc
-  version: 0.1.0.dev2
+  version: 0.8.0
 
 source:
   git_url: https://github.com/xnd-project/rbc.git
-  git_rev: v0.1.0.dev2
+  git_rev: v0.8.0
   #git_url: ../
 
 test:
   requires:
     - pytest
   commands:
-    - pytest -sv -r s --pyargs rbc
+    - pytest -sv -r s -k "not test_python_version" --pyargs rbc
   imports:
     - rbc
     - rbc.thrift
 
 requirements:
-  build:
-    - python
-    - setuptools
   host:
+    - cffi
     - python
-    - pytest-runner
-    - pytest
-    - thriftpy2
-    - tblib
-    - llvmlite >=0.29
-    - numba
-    - netifaces
+    - setuptools
+    - pip
   run:
+    - cffi
     - python
     - thriftpy2
     - tblib
-    - llvmlite
-    - numba
-    - six
+    - llvmlite >=0.29
+    - numba >=0.52
```

RBC doesn't support numba < 0.54
```suggestion
    - numba >=0.54
```